ddp-workspace/projects/ddp-sdk/src/lib/services/irbPassword.service.spec.ts

```diff
@@ -8,29 +9,41 @@ import { LoggingService } from './logging.service';
 describe('IrbPasswordService', () => {
     let service: IrbPasswordService;
     let httpClient: HttpClient;
+    let httpTestingController: HttpTestingController;
     const config = new ConfigurationService();
-    config.backendUrl = 'https://pepper-dev.datadonationplatform.org';
-    // config.backendUrl = 'http://localhost:5555';
-    config.studyGuid = 'ANGIO';
+    config.backendUrl = 'https://pepper-mock-backend.org';
+    config.studyGuid = 'study';
+    const mockRequestUrl = `${config.backendUrl}/pepper/v1/studies/${config.studyGuid}/irb-password-check`;
     // Faking out the cookieservice
     const cookieServiceSpy: jasmine.SpyObj<CookieService> = jasmine.createSpyObj('CookieService', ['put']);
     // called within the parent class
     const loggingServiceSpy: jasmine.SpyObj<LoggingService> = jasmine.createSpyObj('LoggingService', ['logException']);
+
     beforeEach(() => {
         TestBed.configureTestingModule({
             // this the import to get a real httpclient
-            imports: [HttpClientModule],
-            providers: [IrbPasswordService, { provide: CookieService, useValue: cookieServiceSpy },
-                { provide: 'ddp.config', useValue: config }, LoggingService]
+            imports: [
+                HttpClientModule,
+                HttpClientTestingModule
+            ],
+            providers: [
+                IrbPasswordService,
+                CookieService,
+                { provide: 'ddp.config', useValue: config },
+                LoggingService
+            ]
         });
         // For some reason TestBed is willing to inject Configuration into service, but not httpclient
         // So we need to instantiate ourselves
-        httpClient = TestBed.get(HttpClient);
+        httpClient = TestBed.inject(HttpClient);
```

Comment:
In Angular 9+ we should use `TestBed.inject` instead of `TestBed.get`, `TestBed.inject` is depricated.